docs/api/expect.md

```diff
@@ -27,6 +27,36 @@ type Awaitable<T> = T | PromiseLike<T>
 `expect` has no effect on testing types, if the expression doesn't have a type error. If you want to use Vitest as [type checker](/guide/testing-types), use [`expectTypeOf`](/api/expect-typeof) or [`assertType`](/api/assert-type).
 :::
 
+## soft
+
+- **Type:** `ExpectStatic & (actual: any) => Assertions`
+
+`expect.soft` functions similarly to `expect`, but instead of terminating the test execution upon a failed assertion, it continues running and marks the failure as a test failure. All errors encountered during the test will be displayed until the test is completed.
+
+  ```ts
+  import { expect } from 'vitest'
+
+  test('expect.soft test', () => {
+    expect.soft(1 + 1).toBe(3) // mark the test as fail and continue
+    expect.soft(1 + 2).toBe(4) // mark the test as fail and continue
+  })
+  // At the end of the test, the above errors will be output.
+  ```
+  
+  it can also be used with `expect`. if `expect` assertion fails, the test will be terminated and all errors will be displayed.
+
+  ```ts
+  import { expect } from 'vitest'
+
+  test('expect.soft test', () => {
+    expect.soft(1 + 1).toBe(3) // mark the test as fail and continue
+    expect(1 + 2).toBe(3) // failed and terminate the test, all previous errors will be output
+    expect.soft(1 + 2).toBe(4) // do not run
+  })
+  ```
+  
+  > **Note:** `expect.soft` only supported run inside the [test](/api/#test) function.
```

Comment:
We use `::: note` syntax for notes and warnings

And this looks more like a warning to me